sample-apps/react-native/dogfood/package.json

```diff
@@ -25,7 +25,7 @@
     "@stream-io/video-filters-react-native": "workspace:^",
     "@stream-io/video-react-native-sdk": "workspace:^",
     "axios": "^1.8.1",
-    "react": "19.0.0",
+    "react": "19.2.0",
```

Comment:
Won't we run into compatibility issues now that RN 0.79.2 expects React 19.0.0? Same for the other RN sample apps.